### DISCUSSION TITLE
bat key bindings

### DISCUSSION THREAD

**Discussion by alpineyahoo** [sharkdp/bat#2346] [Q&A]
I accidentally hit "s" key when I was using bat in pager mode. I wanted to hit d to scroll down. However, even though I hit the s key the pager was normally scrolled down. Which pager does bat use? It seems to be "less" pager, but less pager seems to lack s key binding. Why do you think s key worked?

**Comment by sharkdp** [comment]
Yes, we use the `less` pager by default, but you can configure that (via `PAGER`, `BAT_PAGER`, the `--pager` option or via bats config file). `s` is a default key binding in less (see `man lesskey`):
>             s          toggle-option o
